client/src/config.js

```diff
@@ -0,0 +1 @@
+export const API_URL = "http://localhost:3000/api";
```

Comment:
Does Heroku also use localhost:3000 as well? 🙈

Reply:
No it doesn't, but there's no way to provide env variables to the react app because it's being served as a static resource through express (the starter kit we are using is setup this way)

Not sure how to solve this exactly, but and idea is to make a request to the server which can tell the react app if it's prod/test/local